perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -257,8 +261,19 @@ def RemoteCommand(self, command,
                     should_log=False, retries=SSH_RETRIES,
                     ignore_failure=False, login_shell=False,
                     suppress_warning=False):
+    return self.RemoteHostCommand(command, should_log, retries,
+                                  ignore_failure, login_shell,
+                                  suppress_warning)
+
+  def RemoteHostCommand(self, command,
```

Comment:
This would lead to a lot of repeated code though... For example I'd copy paste the entirety of BaseLinuxMixin.RemoteCopy into ContainerizedDebianMixin.RemoteHostCopy
Same thing with the methods listed above.

Do you really think this is the best approach? If so, I can go ahead and make the changes.